api/src/uvisor-input.S

```diff
@@ -106,6 +106,9 @@ uvisor_config:
     /* Privileged system IRQ hooks */
     .long __uvisor_priv_sys_irq_hooks
 
+    /* uVisor Lib Functions for use by uVisor in unprivileged mode */
```

Comment:
The comment is for this entire section, not just the one function. This function is called by uVisor in unprivileged mode; this function is provided by uVisor Lib; I'm not sure what is misleading.